nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -603,7 +603,7 @@ def drop_agents(self):
                 self.logger.debug(f"{self.my_info.fqcn}: removing agent {a}")
                 self.agents.pop(a, None)
 
-    def create_internal_listener(self):
+    def make_internal_listener(self):
```

Comment:
Thanks for catching the name conflict between the method name and instance variable name!